test/core/address_utils/parse_address_test.cc

```diff
@@ -56,16 +58,16 @@ static void test_grpc_parse_unix_abstract(const char* uri_text,
   absl::StatusOr<grpc_core::URI> uri = grpc_core::URI::Parse(uri_text);
   if (!uri.ok()) {
     gpr_log(GPR_ERROR, "%s", uri.status().ToString().c_str());
-    GPR_ASSERT(uri.ok());
+    ASSERT_TRUE(uri.ok());
   }
   grpc_resolved_address addr;
 
-  GPR_ASSERT(1 == grpc_parse_uri(*uri, &addr));
+  ASSERT_EQ(1, grpc_parse_uri(*uri, &addr));
   struct sockaddr_un* addr_un =
       reinterpret_cast<struct sockaddr_un*>(addr.addr);
-  GPR_ASSERT(AF_UNIX == addr_un->sun_family);
-  GPR_ASSERT('\0' == addr_un->sun_path[0]);
-  GPR_ASSERT(0 == strncmp(addr_un->sun_path + 1, pathname, strlen(pathname)));
+  ASSERT_EQ(AF_UNIX, addr_un->sun_family);
+  ASSERT_EQ('\0', addr_un->sun_path[0]);
+  ASSERT_EQ(0, strncmp(addr_un->sun_path + 1, pathname, strlen(pathname)));
```

Comment:
This could be expressed better as `ASSERT_TRUE(absl::StartsWith(...`

Reply:
Done